app/controllers/state_file/submission_pdfs_controller.rb

```diff
@@ -0,0 +1,31 @@
+module StateFile
+  class SubmissionPdfsController < ApplicationController
+    def show
+      @submission = EfileSubmission.find_by(id: params[:id])
```

Comment:
Let's wrap this in a `current_client.find(...)` or `current_intake.find(...)` or similar, to ensure we're only looking at the current client's data.

I took a look at how to do that, and I got a little confused about the best way to do it. Maybe it does make sense to subclass the questions controller.

Reply:
I wrote a test locally covering that case, but I didn't push it to GitHub yet because I wanted to start by seeing how we handle other scenarios, and I figured I could turn my ideas into GitHub comments.